server/src/infra/tracing.ts

```diff
@@ -0,0 +1,50 @@
+import { AsyncLocalStorageContextManager } from '@opentelemetry/context-async-hooks';
+import { ExportResult } from '@opentelemetry/core';
+import { PrometheusExporter } from '@opentelemetry/exporter-prometheus';
+import { HttpInstrumentation } from '@opentelemetry/instrumentation-http';
+import { IORedisInstrumentation } from '@opentelemetry/instrumentation-ioredis';
+import { NestInstrumentation } from '@opentelemetry/instrumentation-nestjs-core';
+import { PgInstrumentation } from '@opentelemetry/instrumentation-pg';
+import { NodeSDK } from '@opentelemetry/sdk-node';
+import { BatchSpanProcessor, ConsoleSpanExporter, ReadableSpan } from '@opentelemetry/sdk-trace-base';
+import { OpenTelemetryModuleOptions } from 'nestjs-otel/lib/interfaces';
+
+class CustomConsoleSpanExporter extends ConsoleSpanExporter {
+  export(spans: ReadableSpan[], resultCallback: (result: ExportResult) => void) {
+    const logs = [];
+    for (const span of spans) {
+      logs.push(`[${span.spanContext().traceId}] ${span.name}: ${span.duration[0] / 1e3 + span.duration[1] / 1e6}ms`);
+    }
+    console.log(logs.join('\n'));
```

Comment:
We should probably use the immich logger for this, right?

Reply:
I thought about this, but I'm not sure how to go about it since the config here lives outside of all the Nest stuff.

Reply:
Oh I see. Then the default logger is probably fine imo, yeah. If not, @jrasm91 will probably state his opinion about this :D